recipes/laserbeamsize/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "laserbeamsize" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/laserbeamsize-{{ version }}.tar.gz
+  sha256: 641ad46358a687a2f3fd5326d600b23e51efbfd1a0bd93db1507435387142b43
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - matplotlib-base
+    - numpy
+    - pillow
+    - python >=3.6
+    - scipy
+    - pytest
+    - requests
+    - imageio
+
+test:
+  imports:
+    - laserbeamsize
+  requires:
+    - pytest
+    - pip
+  source_files:
+    - tests/test_no_noise.py
+  commands:
+    - pip check
+    - pytest tests/test_no_noise.py
+
+about:
+  home: https://github.com/scottprahl/laserbeamsize.git
+  summary: |
```

Comment:
This summary is quite long. Do you think you can shorten it?

Reply:
shortened.